packages/discovery/src/location.js

```diff
@@ -0,0 +1,20 @@
+import { Schema, provide, capability as createCapability } from '@ucanto/server'
+
+/**
+ * location claim
+ */
+export const capability = createCapability({
+  can: 'discovery/assert/location',
+  with: Schema.did(),
+  nb: Schema.struct({}),
+})
+
+export const invoke = async () => {
+  return {
+    ok: {},
+  }
+}
+
+export const provider = provide(capability, invoke)
```

Comment:
🤔 because inovoke above has no annotation and `provide` here does not have `invoke` inline it could mislead TS inference. When you inline TS inference will consider `capability` type to infer `invoke` however because `invoke` is already defined TS had inferred it and will no longer update type info for it. I don't know if it would affect `provider` type in this instance, but that is a possibility.